### ISSUE TITLE
[Icon request] Add an icon for visual studio team services

### ISSUE THREAD

**Issue by totkeks** [vscode-icons/vscode-icons#1632]
- [x] I'm submitting an icon request:
  - Type: `folder`
  - Icon Name: `vsts`
  - Sample original Icon: ![vsts](https://user-images.githubusercontent.com/2941389/41596976-6102b51c-73cc-11e8-89d7-76c99ebea970.png)
  - Extensions: `none`
  - Folder names: `.vsts`, `vsts`
  - Language ids: `none`

- More info:
  - [Icon as zipped SVG](https://github.com/vscode-icons/vscode-icons/files/2115132/vsts.zip) (was not allowed to upload SVG, so I had to zip it)

  - basically this is supposed to be used like `docker` or `.docker`, put all your CI/CD related files in there instead of having them clutter your root directory

**Comment by JimiC** [comment]
Please see #650.
